data/json/items/armor.json

```diff
@@ -3008,7 +3008,7 @@
     "type": "ARMOR",
     "name": "pair of eclipse glasses",
     "name_plural": "pairs of eclipse glasses",
-    "description": "A pair of cardboard-framed glasses with UV and IR-filtered lenses topped with a layer of aluminum coating.  They allow safely staring at extremely bright and harmful sources of light like the sun during a solar eclipse.",
+    "description": "A pair of cardboard-framed glasses with UV and IR-filtered lenses topped with a layer of aluminum coating.  They allow safe viewing of extremely bright and harmful sources of light, like the sun.",
```

Comment:
I think "during a solar eclipse" should not be removed since that's basically what eclipse glasses are for.

Reply:
I was thinking that these glasses allow viewing of the sun *regardless* of whether there's an ongoing eclipse or not. The old sentence implied that they were only used when the sun's intensity was mitigated by an eclipse, which is false. I'm ambivalent about it, looking in hindsight, so I'll revert.